src/internal/components/pluggableVisualizations/geoChart/tests/PluggableGeoPushpinChart.spec.tsx

```diff
@@ -1,9 +1,11 @@
 // (C) 2019-2020 GoodData Corporation
 import noop = require("lodash/noop");
+import get = require("lodash/get");
```

Comment:
don't overuse lodash.get
It complicates type control

Reply:
updated